tests/Integration/RawData/RawDataParserTest.php

```diff
@@ -119,4 +119,23 @@ public function testDecodeObjectHeaderIssue405()
 
         $this->assertStringContainsString('Bug fix: PR #405', $pages[0]->getText());
     }
+
+    /**
+     * Tests buggy behavior of decodeXrefStream.
+     *
+     * When PDF has more than one entry in the /Index area (for example by changing the document description), only the first entry is used.
+     *
+     * @see https://github.com/smalot/pdfparser/pull/479
```

Comment:
What exactly would happen in your test if the fix is not used?

Reply:
If the fix is not used the array returned by getDetails() contains only the entry with the key 'Pages'. All other entries like 'Author', 'Creator', 'Title', 'Subject' (which come from the 'Info' object) are not listed, because the 'Info' object gets a wrong object id during parsing the data into the xref structure. So the object id listed at the /Info entry is not valid and the data of the info object cannot be loaded during executing Document::buildDetails().

Reply:
* [ ] Can you add this to the function header too? It helps a lot.

* [ ] Also, can you check for all expected keys not just `Subject`?

Reply:
I added the rest of the keys, but they available only for this PDF.Other PDFs may have more or less keys.

Question: Several tests fail, but it seems that my code isn't causing it. Is that ok?

Reply:
> Question: Several tests fail, but it seems that my code isn't causing it. Is that ok?

No, unfortunately not.